### PR TITLE
Add device inspectors

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Adds slave device inspector.
Allows for more convenient targeting of all* multitool-linkable master devices by providing `Closer` / `Further` buttons.
Includes the gizmo to view the connections.

- Adds master device inspector for multitool-linkable master devices to view connections.
- `Device Linker` no longer automatically reattempts to reconnect distant devices; a button is provided instead.

> *Some multitool-connectable device scripts might need some work to take advantage of the inspectors.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Editor/Inspector/SlaveDeviceInspector.cs [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+using Objects.Other;
+using Systems.Electricity;
+
+
+namespace CustomInspectors
+{
+	/// <summary>
+	/// <para>Allows the mapper to conveniently link a device to an <see cref="AirController"/>, and find disconnected devices.</para>
+	/// <remarks>See also <seealso cref="ACUDevice"/>.</remarks>
+	/// </summary>
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Core/Editor/Inspector/SlaveDeviceInspector.cs]:
```suggestion
	/// <summary>
	/// <para>Allows the mapper to conveniently link a slave device to a master device, and find disconnected slave devices.</para>
	/// </summary>
```

**Comment by corp-0** [comment]
Testing this in editor right now. Seems like all devices in TestStation lost connection to APC, not sure if caused by a change here or what

**Comment by corp-0** [comment]
Fallstation has doors without... wifi? 
image

**Comment by corp-0** [comment]
the mass device linker could have an undo button, not a request but more like a suggestion for the future.

**Comment by NoooneyDude** [comment]
> Fallstation has doors without... wifi?
> image

Lol. They may not be connected to a fire alarm. As for Test Station, not sure, sounds interesting.

**Comment by corp-0** [comment]
got an NRE after changing scene to Outpost and trying the device linker 
```
MissingReferenceException: The object of type 'APCPoweredDevice' has been destroyed but you are still trying to access it.
Your script should either check if it is null or you should not destroy the object.
```

Changing to yet another scene keeps the same amount of "unlinked devices" which makes me think the tool didn't load the new scene.

**Comment by corp-0** [comment]
oh yeah, self powered devices give you the warning and show the gizmo icon as well 
image

**Comment by NoooneyDude** [comment]
Okay, issues addressed.
- Fix Device Linker NRE, not updating after scene change.
- Fix self-sufficient devices showing warning and gizmo by adding new property `RequireLink`.

For FallStation issue, yep, firelocks without a corresponding fire alarm connection under the airlocks.
For TestStation, only three APCs weren't self sufficient. I've updated Device Linker to make this a little more clearer.

Thanks for taking the time to test, Gilles.

**Comment by corp-0** [comment]
I will test it later today

**Review by Bod9001 — APPROVED** [review]

**Comment by corp-0** [comment]
just tested, it seems to be working as intended.

**Review by corp-0 — APPROVED** [review]

**Pull request merged**